blocks/init/src/blocks/components/copyright/docs/readme.md

```diff
@@ -0,0 +1,35 @@
+# Copyright Component
+
+A component used for rendering site's copyright.
+
+## Dependencies
+
+None
+
+## Attributes
+
+* _@param_ **blockClass** | `string` | default: `"header"` | Component's class. If modifying make sure the provide the default class as well (see example call).
+* _@param_ **by** | `string` | default: `"Infinum"` | Name of copyright holder. Generally should be your website / company's name.
+* _@param_ **year** | `int` | default: `gmdate( 'Y' ) (current year)` | Year of the copyright, defaults to current year.
+
+## Example call
+
+```php
+use Eightshift_Libs\Blocks\Helpers\Components;
+
+Components::render( 'copyright' );
+
+// or... 
+
+Components::render( 'copyright', [
+  'by' => 'Your site name',
+] );
+```
+
+## Implementation
+
+1. Copy/Paste component folder in your project..
+2. Rename `eightshift-boilerplate` namespace to your project's namespace..
+3. Add or remove features you are going to use.
```

Comment:
> Add or remove features you are going to use.

Add or remove features you are/aren't going to use.

Reply:
Generally I'd agree but this section is the same on all components so it doesn't really make sense just to change it here. I'd rather if we reviewed it and then update in all components in a single PR. 🤷‍♂